Android/Nothing/fastlane/report.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<testsuites>
+  <testsuite name="fastlane.lanes">
+    
```

Comment:
Remove this empty lines, please

Reply:
This file shouldn't be here. I'll deleted and add it to gitignore on a different PR